sdks/python/setup.py

```diff
@@ -79,7 +93,7 @@ def get_version():
         'apache_beam/utils/counters.py',
     ]),
     install_requires=REQUIRED_PACKAGES,
-    package_data={'': ['*.pyx', '*.pxd']},
+    package_data={'apache_beam': ['*.pyx', '*.pxd', 'pom.xml']},
```

Comment:
An easy way to test this kind of change (I never know if I get setup.py right :-) is to create a fresh virtual env and then run a python setup.py test -- does this land the Cython files and pom file int he proper places?